apps/web/src/pages/get-started/components/CreateWorkflowButton.tsx

```diff
@@ -0,0 +1,73 @@
+import { useQuery } from '@tanstack/react-query';
+import styled from '@emotion/styled';
+
+import { colors } from '@novu/design-system';
+
+import { ROUTES } from '../../../constants/routes.enum';
+import { useCreateTemplateFromBlueprint } from '../../../api/hooks';
+import { errorMessage } from '../../../utils/notifications';
+import { getBlueprintTemplateById } from '../../../api/notification-templates';
+import { getWorkflowBlueprintDetails } from '../../../utils';
+import { TemplateCreationSourceEnum } from '../../templates/shared';
+import { useSegment } from '../../../components/providers/SegmentProvider';
+
+export function CreateWorkflowButton({
+  blueprintIdentifier,
+  children,
+}: {
+  blueprintIdentifier: string;
+  children: string;
+}) {
+  const openNewTab = (route: ROUTES, param: string) =>
+    window.open(`${window.location.origin}${route.replace(':templateId', param)}`, '_blank', 'noreferrer noopener');
+
+  const { createTemplateFromBlueprint } = useCreateTemplateFromBlueprint({
+    onSuccess: (template) => {
+      openNewTab(ROUTES.WORKFLOWS_EDIT_TEMPLATEID, template._id ?? '');
+    },
+    onError: () => {
+      errorMessage('Something went wrong while creating template from blueprint, please try again later.');
+    },
+  });
+
+  const { refetch } = useQuery(
+    ['blueprint', blueprintIdentifier],
+    () => getBlueprintTemplateById(blueprintIdentifier as string),
+    {
+      enabled: false,
+      refetchOnWindowFocus: false,
+
+      onSuccess: (data) => {
+        if (data) {
+          const { name: blueprintName } = getWorkflowBlueprintDetails(data.name);
+          createTemplateFromBlueprint({
+            blueprint: { ...data, name: blueprintName },
+            params: { __source: TemplateCreationSourceEnum.ONBOARDING_GET_STARTED },
+          });
+        }
+      },
+    }
+  );
+
+  const segment = useSegment();
+  const handleCreateTemplateClick = () => {
+    segment.track('[Template Store] Click Create Notification Template', {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Is this an accurate tracking event (in reference to `[Template Store]`)?‏

Reply:
Good question thanks for bringing it up, i was totally not sure about the events here, i left them so i could comment on them but in the end forgot to 🤦‍♂️

Reply:
I wonder what prefix should we keep here because we create the workflow by `Template Store` and we have the location `location: TemplateCreationSourceEnum.ONBOARDING_GET_STARTED`.
@antonjoel82  wdyt?

Reply:
I would say that we're never in the template store though, so this would mis-represent the source of the action in segment IMO. I would say `[Get Started]` may be a better fit

Reply:
Done